### DISCUSSION TITLE
Is useSWRMutation not cached?

### DISCUSSION THREAD

**Discussion by vaynevayne** [vercel/swr#2775] [Q&A]
When I go back to the previous paging, instead of applying the previous cache, I query it each time

**Comment by Brandon-Ln** [comment]
I have the same confusion. As I mentioned in this issue: https://github.com/vercel/swr/issues/2267#issuecomment-1633021360

The current `useSWRMutation`'s default behavior seems to not share the storage state in memory which differentiates it from `useSwr`. This makes it a bit difficult for me to to share the server response data between multi-level depth components. 

Through the issue link before, I found that my purpose could be fullfilled by encapsulating both `useSwr` and `useSwrMutation` at the same hook, but it still feels a little hacky to me, especially in these common use cases. 

So, I want to know what the official recommended practice is in this scenario. Is there anyone who can answer? I would really appreciate it.
